docs/sources/developers/plugins/migration-guide.md

```diff
@@ -4,77 +4,451 @@ title = "Plugin migration guide"
+  log.DefaultLogger.Info("CheckHealth called", "request", req)
+  // The main use case for these health checks is the test button on the
+  // datasource configuration page which allows users to verify that
+  // a datasource is working as expected.
+}
+```
+
+### Unsigned backend plugins will not be loaded
+
+Before you can run a backend plugin on Grafana installations, you must sign the plugin. Grafana Labs strongly recommends you upload signed plugins since we cannot guarantee the integrity of unsigned plugins.
+
+For instructions on plugin signing, refer to the [Plugin signatures](https://grafana.com/docs/grafana/latest/developers/plugins/sign-a-plugin/#sign-a-plugin) topic.
+
+You can still run and develop an unsigned plugin by running your Grafana instance in [development mode](https://grafana.com/docs/grafana/latest/administration/configuration/#app_mode).
+
+### Time series data can now be structured in a wide or many format.
+
+In Grafana 8.x, you can structure time series data in either `wide` or `many` formats. If your plugin supports time series data, we recommend that you support both formats.
+
+Grafana supported only the `many` format before. For example:
+
+| time                 | temperature |
+| -------------------- | ----------- |
+| 2021-05-08T07:31:45Z | 23          |
+| 2021-05-08T09:31:45Z | 25          |
+| 2021-05-08T11:31:45Z | 27          |
 
-It's written for:
+| time                 | humidity |
+| -------------------- | -------- |
+| 2021-05-08T07:31:45Z | 60       |
+| 2021-05-08T09:31:45Z | 55       |
+| 2021-05-08T11:31:45Z | 55       |
 
-- Plugin authors who want to migrate their plugins to Grafana 7.0+.
-- Plugin users who are using custom plugins and want to know whether they can upgrade to Grafana 7.0 without losing functionality.
+> Data is delivered as an array of data frames where the `time` field is repeated.
 
-> If you've successfully migrated your plugin from Angular to React, please [submit an issue on GitHub](https://github.com/grafana/grafana/issues/new?title=Docs%20feedback:%20/developers/plugins/migration-guide.md) and share your experiences with us so that we can improve this guide!
+That made it possible to detect time series data by inspecting the data frame. If it had two fields (time + value) it most likley was time series data. This is no longer possible since the same data can be delivered in the following `wide` format.
 
-## What's new in Grafana 7.0?
+| time                 | temperature | humidity |
+| -------------------- | ----------- | -------- |
+| 2021-05-08T07:31:45Z | 23          | 60       |
+| 2021-05-08T09:31:45Z | 25          | 55       |
+| 2021-05-08T11:31:45Z | 27          | 55       |
+
+> Data delivered as one single data frame (the values share the same time field).
+
+If your plugin should support time series data we recommend you to make sure to support both.
+
```

Comment:
Why would I support multiple time series formats? What's the point of that?

_How_ do I support multiple formats? Can I just return all formats from my data source and be done with it?

Reply:
That is basically target to panel plugins. The reason why we need to suport both `wide` and `many` is because depending on the data source you will receive time series data in one of the formats. So if you want to support all different kinds of data sources we recommend you to handle time series data in both the wide and many format.

Reply:
I think that entire comment could be in these docs to provide context

 - this is for panel plugins
 - why: because data sources can return both formats, and for your panel to support all data sources it should support both wide and many.

Reply:
Good suggestions! 

After discussing this with the team shortly **I think we are removing the time-series part from the migration guide for now.**

The reason being is that we need a bit more time to understand motivations behind these changes, and then put together something that clearly explains who should be concerned about this change, and what are the actual steps that need to be followed in certain scenarios.

Thanks a lot for the feedback, we will link in the comments in the upcoming separate PR.